content/source/docs/enterprise/before-installing/reference-architecture/aws.md

```diff
@@ -90,6 +90,8 @@ allows for further [server-side
 encryption](https://docs.aws.amazon.com/AmazonS3/latest/dev/serv-side-encryption.html)
 by S3 if required by your security policy.
 
+-> **Note:** Terraform Enterprise does not require S3 Versioning feature, however it is recommended to enable S3 Versioning for the backup purpose. The Terraform Enterprise also have routine jobs to delete the expired objects from the S3 storage. There are also certain operations that destroy database records and the associated storage objects, therefore the database backup restoration in the failure scenario fundamentally require the snapshot of the storage objects from the same point in time.
```

Comment:
See what you think of this! I tried to make it a lot shorter. But please let me know if I've fundamentally changed the meaning. 

```suggestion
-> **Note:** Terraform Enterprise has routine jobs that delete expired objects from S3 storage and operations that destroy database records and the associated storage objects. We recommend enabling S3 Versioning so that you will have regular snapshots that you can use to restore your database if necessary.
```